op-node/p2p/gossip.go

```diff
@@ -140,12 +143,21 @@ func BuildGlobalGossipParams(cfg *rollup.Config) pubsub.GossipSubParams {
 	return params
 }
 
-func NewGossipSub(p2pCtx context.Context, h host.Host, cfg *rollup.Config, gossipConf GossipSetupConfigurables, m GossipMetricer) (*pubsub.PubSub, error) {
+// NewGossipSub configures a new pubsub instance with the specified parameters.
+// PubSub uses a GossipSubRouter as it's router under the hood.
+func NewGossipSub(p2pCtx context.Context, h host.Host, g ConnectionGater, cfg *rollup.Config, gossipConf GossipSetupConfigurables, m GossipMetricer) (*pubsub.PubSub, error) {
 	denyList, err := pubsub.NewTimeCachedBlacklist(30 * time.Second)
 	if err != nil {
 		return nil, err
 	}
 	params := BuildGlobalGossipParams(cfg)
+	// TODO: make this configurable behind a cli flag - disabled or default
+	peerScoreParams, err := GetPeerScoreParams("default")
```

Comment:
Some score params will probably depend on rollup config properties like the block time. Would make sense to add the config as argument.